calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -2,22 +2,17 @@
 description: Install NodeLocal DNSCache
 ---
 
-# Install NodeLocal DNSCache
+# Use NodeLocal DNSCache in your cluster
 
 ## Big picture
```

Comment:
Suggest the following:

## Big picture
Set up NodeLocal DNSCache to improve DNS lookup latency. 

## Before you begin

**Required**
Follow these [steps](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to enable NodeLocal DNSCache connectivity.

**Unsupported**
- OpenShift
    NodeLocal DNSCache does not support OpenShift. Openshift has its own DNS caching mechanism which deploys CoreDNS pods as DaemonSets.
For more information, see [Openshift DNS setup](https://docs.openshift.com/container-platform/4.10/networking/dns-operator.html).